vec120-assertions/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
why extra gitignore files? This should be included in the file at root level (maybe with **/target/*)

Reply:
I just stayed with the "conventions" of the project. See `/vec113/.gitignore` and `/vec120/.gitignore` ;-)